### ISSUE TITLE
Server disconnects when creating a Lesson

### ISSUE THREAD

**Issue by lauradanforth** [learningequality/kolibri#8623]
## Observed behavior
After clicking on New Lesson, the server disconnected and wasn't able to reconnect.

…

## Errors and logs

From the console:
```
11Failed to decode downloaded font: <URL>
11OTS parsing error: glyf: zero-length table
kolibri.core.default_frontend-0.15.0b1.js:1 Kolibri Modules: kolibri.plugins.coach.app registered
kolibri.core.default_frontend-0.15.0b1.js:1 Request error: undefined, 0 for GET to /coach/api/notifications/ - open for more info
:8080/coach/api/notifications/?classroom_id=3c1c95d4e24f44cece814c80e8043443&after=0&1636152337445=1636152337445:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
kolibri.core.default_frontend-0.15.0b1.js:1 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'method')
    at Resource.value (kolibri.core.default_frontend-0.15.0b1.js:1)
    at kolibri.core.default_frontend-0.15.0b1.js:1
kolibri.core.default_frontend-0.15.0b1.js:1 [ERROR: kolibri/core/assets/src/heartbeat.js] Session polling failed, with error:  Error: Network Error
    at module.exports (kolibri.core.default_frontend-0.15.0b1.js:1)
    at XMLHttpRequest.request.onerror (kolibri.core.default_frontend-0.15.0b1.js:1)
(anonymous) @ kolibri.core.default_frontend-0.15.0b1.js:1
:8080/api/auth/session/current/:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
kolibri.core.default_frontend-0.15.0b1.js:1 [ERROR: kolibri/core/assets/src/heartbeat.js] Session polling failed, with error:  Error: Network Error
    at module.exports (kolibri.core.default_frontend-0.15.0b1.js:1)
    at XMLHttpRequest.request.onerror (kolibri.core.default_frontend-0.15.0b1.js:1)
(anonymous) @ kolibri.core.default_frontend-0.15.0b1.js:1
:8080/api/auth/session/current/:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
kolibri.core.default_frontend-0.15.0b1.js:1 [ERROR: kolibri/core/assets/src/heartbeat.js] Session polling failed, with error:  Error: Network Error
    at module.exports (kolibri.core.default_frontend-0.15.0b1.js:1)
    at XMLHttpRequest.request.onerror (kolibri.core.default_frontend-0.15.0b1.js:1)
(anonymous) @ kolibri.core.default_frontend-0.15.0b1.js:1
:8080/api/auth/session/current/:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
kolibri.core.default_frontend-0.15.0b1.js:1 [ERROR: kolibri/core/assets/src/heartbeat.js] Session polling failed, with error:  Error: Network Error
    at module.exports (kolibri.core.default_frontend-0.15.0b1.js:1)
    at XMLHttpRequest.request.onerror (kolibri.core.default_frontend-0.15.0b1.js:1)
(anonymous) @ kolibri.core.default_frontend-0.15.0b1.js:1
:8080/api/auth/session/current/:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
DevTools failed to load source map: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/content.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.user_profile.user_profile_side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.coach.side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.facility.side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.device.side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.learn.side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.user_auth.login_side_nav-0.15.0b1.js.map: Load canceled due to load timeout
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/static/kolibri.plugins.coach.app/kolibri.plugins.coach.app-0.15.0b1.js.map: Load canceled due to load timeout
```
-->

…

## Expected behavior

…

## User-facing consequences

…

## Steps to reproduce
While logged in as coach or admin, attempt to create a Lesson

…

## Context

Tell us about your environment, including:
 * 0.15
 * Mac OS (.pex file)
 * Chrome
![image](https://user-images.githubusercontent.com/22602993/140587163-833cc669-0541-45e0-9307-7fa69d5eae4b.png)

…

**Comment by lauradanforth** [comment]
Update, this is also happening later in the lesson creation process. While searching for resources:

```28Failed to decode downloaded font: <URL>
28OTS parsing error: glyf: zero-length table
kolibri.core.default_frontend-0.15.0b1.js:sourcemap:1 Kolibri Modules: kolibri.plugins.coach.app registered
kolibri.core.default_frontend-0.15.0b1.js:sourcemap:1 Request error: undefined, 0 for GET to /coach/api/classsummary/3c1c95d4e24f44cece814c80e8043443/ - open for more info
:8080/coach/api/classsummary/3c1c95d4e24f44cece814c80e8043443/?1636153783851=1636153783851:1 Failed to load resource: net::ERR_CONNECTION_RESET
:8080/en/coach/#/3c1…c4d7318/selection:1 Uncaught (in promise) Cancel
kolibri.core.default_frontend-0.15.0b1.js:sourcemap:1 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'method')
    at Resource.value (kolibri.core.default_frontend-0.15.0b1.js:sourcemap:1)
    at kolibri.core.default_frontend-0.15.0b1.js:sourcemap:1
DevTools failed to load source map: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/content.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME```

**Comment by lauradanforth** [comment]
Also happening when creating a Quiz:
```
kolibri.core.default_frontend-0.15.0b1.js:1 Kolibri Modules: kolibri.plugins.coach.app registered
12Failed to decode downloaded font: <URL>
12OTS parsing error: glyf: zero-length table
kolibri.core.default_frontend-0.15.0b1.js:1 Request error: undefined, 0 for GET to /coach/api/notifications/ - open for more info
:8080/coach/api/notifications/?classroom_id=3c1c95d4e24f44cece814c80e8043443&after=0&1636154136501=1636154136501:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
kolibri.core.default_frontend-0.15.0b1.js:1 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'method')
    at Resource.value (kolibri.core.default_frontend-0.15.0b1.js:1)
    at kolibri.core.default_frontend-0.15.0b1.js:1
kolibri.core.default_frontend-0.15.0b1.js:1 [ERROR: kolibri/core/assets/src/heartbeat.js] Session polling failed, with error:  Error: Network Error
    at module.exports (kolibri.core.default_frontend-0.15.0b1.js:1)
    at XMLHttpRequest.request.onerror (kolibri.core.default_frontend-0.15.0b1.js:1)
(anonymous) @ kolibri.core.default_frontend-0.15.0b1.js:1
:8080/api/auth/session/current/:1 Failed to load resource: net::ERR_EMPTY_RESPONSE
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/en/coach/kolibri.plugins.user_profile.user_profile_side_nav-0.15.0b1.js.map: Connection error: net::ERR_CONNECTION_RESET
DevTools failed to load source map: Could not load content for chrome-extension://liecbddmkiiihnedobmlmillhodjkdmb/js/content.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
DevTools failed to load source map: Could not load content for http://127.0.0.1:8080/static/kolibri.core.default_frontend/kolibri.core.default_frontend-0.15.0b1.js.map: HTTP error: status code undefined, net::ERR_EMPTY_RESPONSE```

**Comment by rtibbles** [comment]
I think this is probably the same underlying cause as #8618

**Comment by rtibbles** [comment]
Presumed fixed in #8661
